packages/patternfly-4/react-core/src/components/Modal/Modal.js

```diff
@@ -50,7 +52,8 @@ class Modal extends React.Component {
   static propTypes = propTypes;
   static defaultProps = defaultProps;
 
-  id = `pf-modal-${currentId++}`;
+  bodyId = `pf-modal-${currentId}`;
```

Comment:
Do you know why we got rid of id?

Reply:
I think it is just a rename here.  It is internal so it should be fine